framework/doc/content/syntax/Mesh/splitting.md

```diff
@@ -51,9 +51,20 @@ It is noted that if there are mesh meta data generated by mesh generators, these
 will be written to a binary file under the generated directory that can be loaded when using
 split meshes.
 
+!alert note
+The mesh splitter commands do not work with DistributedMesh. You must only split with a ReplicatedMesh.
+
 ## Using Split Meshes
 
-To use a mesh split configuration use the `--use-split` flag (which takes no arguments):
+There are two ways of using referring to meshes in parallel in MOOSE, as split or as distributed. While there is no conceptual difference between the two there is a difference in implementation and features.
```

Comment:
a split mesh is a mesh read from a split file so far. that's the conceptual difference

Reply:
Not sure it's entirely so given they both handle distributed meshes and there is an overlap, but if you'd like to settle on this difference for MOOSE I am happy with it.

Reply:
it is the difference. If you dont read it from a pre-split file it's not a split mesh